tx_test.go

```diff
@@ -334,8 +334,14 @@ func TestTx_CreateByTableContext(t *testing.T) {
 		tx, err := cache.Begin(txConn)
 		NoError(t, err)
 		defer func() { NoError(t, tx.RollbackUnlessCommitted()) }()
+		if tx.IsCommitted() {
+			t.Fatal("unexpected transaction status")
+		}
 		NoError(t, tx.Commit())
 
+		if !tx.IsCommitted() {
+			t.Fatal("unexpected transaction status")
```

Comment:
I suggest to use `Equal()` or `Equalf()`. 
What do you think it?

Reply:
Thanks!
https://github.com/knocknote/rapidash/pull/6/commits/61d7b3a6cbee8ad559cab9e347a9b77d371ea51b